jvm-packages/xgboost4j/src/test/java/ml/dmlc/xgboost4j/java/BoosterImplTest.java

```diff
@@ -643,7 +643,7 @@ public void testSetAndGetAttrs() throws XGBoostError {
     }});
 
     Map<String, String> attr = booster.getAttrs();
-    TestCase.assertEquals(attr.size(), 4);
+    TestCase.assertEquals(attr.size(), 6);
```

Comment:
@wbo4958 maybe because 2 attributes got added?

Reply:
Yeah, I know the extra 2 attrs, but others may be so confused. Better to add some comments here

Reply:
@wbo4958 Yes as `attr.size()` is 6 because 2 additional attributes are added. The test case is checking setter and getter method and also total number of attributes in the booster object.
I will update the doc to explain best iteration/score. 

Any suggestion on adding unit test for these attribute ?

Reply:
checking if the booster is containing best_iteration and best_score?